src/code/z_eff_ss_dead.c

```diff
@@ -81,14 +81,14 @@ void func_800AE5E4(PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3) {
     CLOSE_DISPS(play->state.gfxCtx);
 }
 
-void func_800AE778(PlayState* play, Color_RGBA8* color, s16 arg2, s16 arg3) {
+void func_800AE778(PlayState* play, Color_RGBA8* color, s16 arg2, s16 duration) {
```

Comment:
Was this on purpose? I'm not sure duration makes sense here.

Reply:
I named it like this because of this in z_actor, then I realized it didn't make much sense but forgot to undo it
```c

            func_800AE778(play, &actorDefaultHitColor, actor->colorFilterTimer,
                          COLORFILTER_GET_DURATION(actor->colorFilterParams));
```